packages/webapp/src/components/Form/Input/index.jsx

```diff
@@ -58,8 +59,20 @@ const Input = ({
     setShowError(false);
   };
 
+  const showPickerHandler = () => {
+    try {
+      input.current.showPicker();
+    } catch (e) {
+      console.log({ e });
+    }
+  };
+
   const onKeyDown = ['number', 'decimal'].includes(type) ? numberOnKeyDown : undefined;
 
+  useEffect(() => {
+    name === 'due_date' ? showPickerHandler() : null;
```

Comment:
It looks like you are creating a new feature on Input with an attribute called autoFocus. I think if you carry forward that option as a boolean here, and scrap the try catch and console.log then you'll have a succinct and reusable feature.

Reply:
I think what Duncan means is:
```
 useEffect(() => {
    if (autoFocus) {
        try {
           input.current.showPicker();
         } catch (e) {
           console.log(e);
         }
  }, []);
```

and I know someone wants to use this feature for a different ticket.

Reply:
I agree with @SayakaOno addressing this requirement in one of the tickets i am working on. There we also need to focus on the input showing the date picker. So we can add this line of code "input.current.focus();":
`useEffect(() => {

    if (autoFocus) {
        try {
           input.current.showPicker();
           input.current.focus();
         } catch (e) {
           console.log(e);
         }
  }, [autoFocus]);`